src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SVContext.java

```diff
@@ -277,9 +280,12 @@ public int getStructuralVariantLength() {
                     composePaddedInterval(contigName, contigLength, start, start, padding));
         } else if (type == StructuralVariantType.DEL) {
             final int end = getEnd();
+            final int homologyPadding = (padForHomology ? getAttributeAsInt(GATKSVVCFConstants.HOMOLOGY_LENGTH, 0) : 0);
             return Arrays.asList(
-                    composePaddedInterval(contigName, contigLength, start, start, padding),
-                    composePaddedInterval(contigName, contigLength, end, end, padding));
+                    composePaddedInterval(contigName, contigLength, start + 1,
```

Comment:
I know this is probably my fault not asking this question before, but is there an off-by-1 error in `composePaddedInterval()`?  it seems to perform an asymmetrical padding to `start` and `end`

Reply:
I think that it is right given that the start of the VariantContext is the position of the base before the event, and you are not allowed to have a zero-length `SimpleInterval`. But I could be wrong about that.

Reply:
Ah, that's right!